gap/FinSetsForCAP.gi

```diff
@@ -4,13 +4,41 @@
 # Implementations
 #
 
-BindGlobal( "FinSets", CreateCapCategory( "FinSets" ) );
-
-SetIsElementaryTopos( FinSets, true );
-
-AddObjectRepresentation( FinSets, IsFiniteSet and HasAsList and HasLength );
+##
+InstallMethod( CategoryOfFinSets,
+               [ ],
+               
+  function ( )
+    local overhead_option, finalize_option, FinSets;
+    
+    overhead_option := CAP_INTERNAL_RETURN_OPTION_OR_DEFAULT( "overhead", true );
+    
+    finalize_option := CAP_INTERNAL_RETURN_OPTION_OR_DEFAULT( "FinalizeCategory", true );
+    
+    FinSets := CreateCapCategory( "FinSets" : overhead := overhead_option );
+    
+    FinSets!.category_as_first_argument := true;
+    
+    SetFilterObj( FinSets, IsCategoryOfFinSets );
+    
+    SetIsElementaryTopos( FinSets, true );
+    
+    AddObjectRepresentation( FinSets, IsFiniteSet and HasAsList and HasLength );
+    
+    AddMorphismRepresentation( FinSets, IsFiniteSetMap and HasAsList );
+    
+    INSTALL_FUNCTIONS_FOR_FIN_SETS( FinSets );
+    
+    if finalize_option then
```

Comment:
Just a note for the future: this is not necessary anymore, since `Finalize` now deals with the option `FinalizeCategory` by itself.

I will fix this here and in `SkeletalFinSets` after the merge.